src/shared/utils/flattenChildren.js

```diff
@@ -54,7 +54,10 @@ function flattenSingleChildIntoContext(
  * children will not be included in the resulting object.
  * @return {!object} flattened children keyed by name.
  */
-function flattenChildren(children: ReactElement<any>, selfDebugID: number): ?{ [name: string]: ReactElement<any> } {
+function flattenChildren(
+  children: ReactElement<any>,
+  selfDebugID?: number,
```

Comment:
Copied from the flow group:
- `(arg?: number)` means `(arg: number | undefined)`
- `(arg: ?number)` means `(arg: number | undefined | null)`

In this case I think they should all be `arg?: number`.